wordpress-mobile/WordPress-iOS
# My Site Dashboard: Hide Quick Actions cell for Jetpack App

## Description

- Fixed an issue where the Quick Actions cell (introduced in [this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/17812)) was being shown for the Jetpack app, even though the cell shouldn't be displayed
- Deleted some obsolete files and code in BlogDetailsVC related to the headerView

Before | After
-- | --
<img src="https://user-images.githubusercontent.com/6711616/152848486-c49607a0-3a95-4993-8221-7c4ada7af626.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/152848474-5e1caeb9-15b7-4946-b885-d430108affdd.png" width=200>

## Notes
- FYI @pachlava -- pinging you since this is a MSD related bug

## How to test
1. Build and run the Jetpack app
2. Log in
3. Go to My Site
4. ✅ The Quick Actions cell should not be displayed

## Regression Notes
1. Potential unintended areas of impact
Quick Actions cell

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested the above steps manually (both Jetpack and WordPress)

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
